docs/reference/groups.yml

```
@@ -130,6 +130,37 @@
     In addition to the functions listed below, the `calc` module also defines
     the constants `pi`, `tau`, `e`, and `inf`.
 
+- name: std
+  title: Standard library
+  category: foundations
+  path: ["std"]
+  details: |
+    A module that contains all globally accessible items.
+
+    This is useful whenever you override a name from the global scope, e.g.,
```

```suggestion
    This is useful whenever you overrode a name from the global scope, e.g.,
```
I think it becomes useful only once the overriding is already in the past.